--- a/psutil/_psutil_aix.c
+++ b/psutil/_psutil_aix.c
@@ -116,8 +116,12 @@
         status.pr_stat = SACTIVE;
     } else {
         sprintf(path, "%s/%i/status", procfs_path, pid);
-        if (! psutil_file_to_struct(path, (void *)&status, sizeof(status)))
-            return NULL;
+        if (! psutil_file_to_struct(path, (void *)&status, sizeof(status))) {
+            // Can't access /proc/<pid>/status (eg: access denied)
+            // Continue without the process status
+            PyErr_Clear();
+            status.pr_stat = 0;
+        }
     }

     return Py_BuildValue("KKKdiiiK",
